app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from typing import Any, Optional
+
+from app.errors import (
+NotVaccinatedError,
+OutdatedVaccineError,
+NotWearingMaskError
+)
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict[str, Any]) -> Optional[str]:
```

Comment:
Code Style Issue: You have used the type hint `dict[str, Any]` for the `visitor` argument of the `visit_cafe` method. It would be better to define a custom class or dataclass for `Visitor` and use it as a type hint instead. This will make your code more readable and maintainable. For instance, you can define a `Visitor` class with `vaccine`, `expiration_date` and `wearing_a_mask` attributes and use it as a type hint for the `visitor` argument.